terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-down.ts

```diff
@@ -349,20 +349,28 @@ export async function getGHRunnerRepo(ec2runner: RunnerInfo, metrics: ScaleDownM
     }
   }
 
-  if (ghRunner === undefined && ec2runner.ghRunnerId !== undefined) {
-    console.warn(
-      `Runner '${ec2runner.instanceId}' [${ec2runner.runnerType}](${repo}) not found in ` +
-        `listGithubRunnersRepo call, attempting to grab directly`,
-    );
-    try {
-      ghRunner = await getRunnerRepo(repo, ec2runner.ghRunnerId, metrics);
-    } catch (e) {
+  if (ghRunner === undefined) {
+    if (ec2runner.ghRunnerId === undefined) {
       console.warn(
-        `Runner '${ec2runner.instanceId}' [${ec2runner.runnerType}](${repo}) error when getRunnerRepo call: ${e}`,
+        `Runner '${ec2runner.instanceId}' [${ec2runner.runnerType}](${repo}) not found in ` +
+          `listGithubRunnersRepo call, and it did not have the GithubRunnerId EC2 tag set.  This ` +
```

Comment:
English is not my native, but `runner could not be found among list of org runners fetch by XYZ API

Reply:
Fair. Tweaked wording